python/ray/autoscaler/v2/scheduler.py

```diff
@@ -1584,6 +1584,12 @@ def _enforce_idle_termination(
                 continue
 
             idle_timeout_s = ctx.get_idle_timeout_s()
+            # Override the scheduler idle_timeout_s if set for this node_type.
```

Comment:
`node_type_configs` is a dictionary. Maybe we can look it up directly instead of iterating through it?

Reply:
Done in [21b12c0](https://github.com/ray-project/ray/pull/48813/commits/21b12c02b5b341370aeb2cefc3fc5c444592be48).